compiler/rustc_middle/src/ty/context.rs

```diff
@@ -877,6 +877,17 @@ impl<'tcx> TyCtxt<'tcx> {
         self.get_lang_items(())
     }
 
+    pub fn default_traits(self) -> &'tcx [rustc_hir::LangItem] {
+        self.get_default_traits(())
```

Comment:
maybe just name the query `default_traits` and remove this helper function? (or document here why it is not done that way)

Reply:
I'm also not sure why it's a query, it returns either one or another `'static` slice depending on a boolean flag, memoization is not useful here.

Reply:
I assumed there was some reason 😆 but yea, from the way it's implemented right now, it's not necessary to be a query